docs/gallery/interactive/fast_fourier_transform.ipynb

```diff
@@ -0,0 +1,353 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## FFT normalizations\n",
+    "Pyfar implements five normalizations  after [Ahrens at al. (2020)](http://www.aes.org/e-lib/browse.cfm?elib=20838) that can be applied to spectra after the DFT. The normalizations are implicitly used by the [pyfar.Signal class](https://pyfar.readthedocs.io/en/stable/classes/pyfar.audio.html#pyfar.classes.audio.Signal), and are available from [pyfar.dsp.fft.normalization](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.fft.html#pyfar.dsp.fft.normalization). For a Signal object `signal`, `signal.freq` contains the normalized spectrum according to `signal.fft_norm` and `signal.freq_raw` contains the raw spectrum without any normalization. **The time data (**`signal.time`**) does not change regardless of the normalization.**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "| Normalization| Equation                                                                                                 |\n",
+    "|-------------|----------------------------------------------------------------------------------------------------------|\n",
+    "| `'none'`    | --                                                                                                       |\n",
+    "| `'unitary'` | $X_{\\text{SS}}(k) = \\left\\{ \\begin{array}{ll} X(k) & \\forall k=0, k=\\frac{N}{2} \\\\ 2 X(k) & \\forall 0<k< \\frac{N}{2} \\end{array} \\right.$|\n",
+    "| `'amplitude'` | $\\overline{X}_{\\text{SS}}(k) = \\frac{1}{N} X_{\\text{SS}}(k)$                                          |\n",
+    "| `'rms'`       | $\\overline{X}_{RMS}(k) = \\left\\{ \\begin{array}{ll} \\frac{1}{\\sqrt{2}} \\overline{X}_{\\text{SS}}(k) & \\forall 0<k< \\frac{N}{2} \\\\ \\quad \\overline{X}_{\\text{SS}}(k) & \\forall k=0, k=\\frac{N}{2} \\end{array} \\right.$|\n",
+    "| `'power'`     | $\\overline{\\overline{X}}_{\\text{SS}}(k) = \\lvert \\overline{X}_{\\text{RMS}}(k) \\lvert ^2$           |\n",
+    "| `'psd'`       | $\\overline{\\overline{\\underline{X}}}_{\\text{SS}}(k) = \\frac{N}{f_s} \\overline{\\overline{X}}_{\\text{SS}}(k) = \\frac{N}{f_s} \\lvert \\overline{X}_{\\text{RMS}}(k) \\lvert ^2$|\n",
+    "\n",
+    "Note that the above formulation holds for real-valued signals with single-sided spectra $X_{\\text{SS}}(k)$. Hence, there are small differences in the definitions compared to the formulas written in [Ahrens et al. (2020)](http://www.aes.org/e-lib/browse.cfm?elib=20838)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example signals\n",
+    "Four signals with a length of 100 samples and a sampling rate of 10 kHz are used for illustrating the normalizations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyfar as pf\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
```

Comment:
```suggestion
    "import matplotlib.pyplot as plt\n",
    "%matplotlib inline\n",
```
to make sure it's plotted in 
https://pyfar-gallery--28.org.readthedocs.build/en/28/gallery/interactive/fast_fourier_transform.html#'none'